ShopifySharp/Entities/ProductListingVariant.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace ShopifySharp
+{
+    public class ProductListingVariant
+    {
+        /// <summary>
+        /// The object's unique id.
+        /// </summary>
+        [JsonProperty("id")]
+        public long Id { get; set; }
+        
+        /// <summary>
+        /// Available current product variant
+        /// </summary>
+        [JsonProperty("available")]
+        public bool Available { get; set; }
+
+        /// <summary>
+        /// Custom properties that a shop owner can use to define product variants.
+        /// </summary>
+        [JsonProperty("option_values")]
+        public OptionValue[] OptionValues { get; set; }
```

Comment:
Prefer to use `IEnumerable<OptionValue>` instead of `OptionValue[]` just for consistency with the rest of the package.

Reply:
Accepted and fix